vue-social/src/config.js

```diff
@@ -1 +1 @@
-export const API_BASE_URL = "http://localhost:8080/";
+export const API_BASE_URL = "http://165.22.182.0:8080/";
```

Comment:
set an env variable, prod and dev

Reply:
also do the same for spring, make a appliction prod yaml file for easy deployment